packages/gtmcore/gtmcore/container/hub_container.py

```diff
@@ -101,6 +101,16 @@ def _dummy_feedback(msg: str) -> None:
                                       f" the build output and fix any issues.\n")
                     raise ContainerException(f"Failed to build the Project image, most likely due to the Project's"
                                              f" environment configuration.")
+                else:
+                    # Request build output from the launch service
+                    feedback_pos = len(build_output)
+                    build_output_resp = requests.get(f"{self._launch_service}/v1/client/{self._client_id}/namespace/{self.labbook.owner}/"
+                                                     f"project/{self.labbook.name}/buildoutput")
+                    if build_output_resp.status_code == 200:
+                        build_output = build_output_resp.json().get("output", "")
+                        feedback_callback(f"{build_output[feedback_pos:]}")
```

Comment:
```suggestion
                        feedback_callback(build_output[feedback_pos:])
```

This should work fine, right?

Reply:
Yeah, updated.